_data/chains/eip155-5555558.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Imversed Testnet",
+  "chain": "Imversed",
+  "rpc": [
+    "http://qt.imversed.com:8545"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Imversed Token",
+    "symbol": "IMV",
+    "decimals": 18
+  },
+  "infoURL": "https://imversed.com",
+  "shortName": "Imversed",
+  "chainId": 5555558,
+  "networkId": 5555558,
+  "icon": "imversed",
+  "explorers": [
+    {
+      "name": "Big Dipper",
+      "url": "https://tex-t.imversed.com",
+      "icon": "brise",
+      "standard": "imversed"
```

Comment:
this is not a valid standard - either use `none` or `EIP3091`

Reply:
Sorry for my stupid mistake and for taking so long to answer. updated

Reply:
@ligi  Hello! 
Maybe something else is wrong? What I can do to make pr merged in master?